rhoshift/main.py

```
@@ -1,36 +1,24 @@
 import sys
 sys.dont_write_bytecode = True
-from utils.operator.cleanup import cleanup
-from cli.args import parse_args
-from cli.commands import install_operator, install_operators
-from logger.logger import Logger
+from rhoshift.utils.operator.cleanup import cleanup
+from rhoshift.cli.args import parse_args
+from rhoshift.cli.commands import install_operator, install_operators
+from rhoshift.logger.logger import Logger
 from typing import Optional
+import pyfiglet
```

Ruff (0.11.9)

`pyfiglet` imported but unused

Remove unused import: `pyfiglet`

(F401)